### PR TITLE
Add Meter, MeterPlus, and Outdoor Meter to SwitchBot Cloud Docs

### PR DESCRIPTION
**Pull request by laurence-presland**
## Proposed change

- Added Meter, MeterPlus, and Outdoor Meter to supported devices list
- Added notice about the SwitchBot API daily limit and the new scan interval config option

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/115522
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/switchbot_cloud.markdown [hunk 1]
```diff
@@ -32,11 +34,18 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
   - ON/OFF for all appliance types excepted "Others"
   - Air Conditioner
+- Sensors (Meter, MeterPlus, Outdoor Meter)
+
+## Important considerations
+
+By default, each individual sensor will poll the SwitchBot Cloud API for a status update every 10 minutes (600 seconds). You can request more frequent updates by lowering the `Scan interval (seconds)` in the initial integration setup.
 
+<div class='note warning'>
+The SwitchBot Cloud API limits users to 10,000 requests per day so please be cautious when lowering the scan interval.
```
Comment (c0ffeeca7) [inline source/_integrations/switchbot_cloud.markdown]:
```suggestion
The SwitchBot Cloud API limits users to 10,000 requests per day. Please be cautious when lowering the scan interval.
```

**Review by SeraphicRav — APPROVED** [review]

File: source/_integrations/switchbot_cloud.markdown [hunk 2]
```diff
@@ -6,10 +6,12 @@ ha_category:
...
   - Remote
   - Switch
+  - Sensor
 ha_release: '2023.10'
 ha_iot_class: Cloud Polling
 ha_codeowners:
   - '@SeraphicRav'
+  - '@laurence-presland'
 ha_domain: switchbot_cloud
 ha_platforms:
```
Comment (joostlek) [inline source/_integrations/switchbot_cloud.markdown:16]: The list of platforms is incomplete

File: source/_integrations/switchbot_cloud.markdown [hunk 3]
```diff
@@ -32,11 +34,18 @@ Please note, device names configured in the SwitchBot app are transferred into H
 
 - Plug (Wi-Fi only, only available in Japan)
 - Plug Mini, both the original and HomeKit-enabled
-
 - IR appliances exposed through the different hubs:
   - ON/OFF for all appliance types excepted "Others"
   - Air Conditioner
+- Sensors (Meter, MeterPlus, Outdoor Meter)
```
Comment (joostlek) [inline source/_integrations/switchbot_cloud.markdown]: Sensors isn't a device. This list is listing all devices

File: source/_integrations/switchbot_cloud.markdown [hunk 4]
```diff
@@ -32,11 +34,18 @@ Please note, device names configured in the SwitchBot app are transferred into H
...
+
+## Important considerations
+
+By default, each individual sensor will poll the SwitchBot Cloud API for a status update every 10 minutes (600 seconds). You can request more frequent updates by lowering the `Scan interval (seconds)` in the initial integration setup.
 
+<div class='note warning'>
+The SwitchBot Cloud API limits users to 10,000 requests per day. Please be cautious when lowering the scan interval.
+</div>
```
Comment (joostlek) [inline source/_integrations/switchbot_cloud.markdown]: This needs rewording

Comment (laurence-presland) [reply source/_integrations/switchbot_cloud.markdown]: Have reworded this. I figure it's good to just give users the information about the request limit. They can do with that what they wish.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @laurence-presland :+1:

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @laurence-presland :+1:

**Pull request merged**